config/tracks/ks_nordschleife.ini

```diff
@@ -859,7 +861,7 @@ CONDITION=NIGHT_SMOOTH
 
 [Material_DistantEmissive]
 DistantGlowRadius_v2=0.6  ;  most influential
-Meshes=NSLIT_adenauerL0, Torus001, NSLITS_litpostglow?, NSLIT_SingleSpotsGlow?, NSLIT_BigSpots?
```

Comment:
Removed Mercedes Logo mesh from distant emissive as it causes a bug in emissive rendering.